.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
these type of changes shouldn't be here... please add ```.vscode``` to ```.gitignore```

Reply:
In a separate commit :)